rkt/run.go

```diff
@@ -313,3 +313,18 @@ func (e *envMap) Strings() []string {
 	}
 	return env
 }
+
+// privateNetList implements the flag.Value interface to allow specification
+// of -private-net with and without values
+type privateNetList string
```

Comment:
Yeah, generally prefer to validate syntactical things ("is this a well-formed list?") as soon as possible, then pass a struct (i.e. in this case probably just a slice of strings) down the chain